src/doc/html_templates/html_templates.nit

```diff
@@ -196,3 +196,71 @@ class DocTopMenu
 		addn "</nav>"
 	end
 end
+
+redef class DocComposite
```

Comment:
Doc to explain the name and the role of the `DocComposite` class because I do not see the *composition* of *doc* part.